src/main/java/team4/footwithme/team/api/TeamMemberApi.java

```diff
@@ -1,11 +1,49 @@
 package team4.footwithme.team.api;
 
 import lombok.RequiredArgsConstructor;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+import team4.footwithme.global.api.ApiResponse;
+import team4.footwithme.team.api.request.TeamMemberRequest;
+import team4.footwithme.team.service.TeamMemberService;
+import team4.footwithme.team.service.response.TeamResponse;
+
+import java.util.List;
 
 @RequiredArgsConstructor
 @RestController
-@RequestMapping
+@RequestMapping("/api/v1/team_member")
```

Comment:
URI에 언더바는 안쓰는걸로 알고있어요 하이푼으로 바꾸거나 하시면 될 것 같아요

```suggestion
@RequestMapping("/api/v1/team-member")
```

[REST API URI 규칙](https://velog.io/@pjh612/REST-API-URI-%EA%B7%9C%EC%B9%99)

이 글 한번 참조해보시면 좋을 것 같아요!

Reply:
아 이거 헷갈리던 부분이였는데 감사합니다!